tools/third_party.py

```diff
@@ -110,6 +141,29 @@ def delete_lockfile():
     delete_lockfile()
 
 
+# Install python packages with pip.
+def run_pip():
+    # Install an recent version of pip into a temporary directory. The version
+    # that is bundled with python is too old to support the next step.
+    temp_python_home = mkdtemp()
+    pip_env = {"PYTHONUSERBASE": temp_python_home}
+    run([sys.executable, "-m", "pip", "install", "--upgrade", "--user", "pip"],
+        cwd=third_party_path,
+        merge_env=pip_env)
+
+    # Install pywin32.
+    run([
+        sys.executable, "-m", "pip", "install", "--upgrade", "--target",
+        python_packages_path, "--platform=win_amd64", "--only-binary=:all:",
+        "pypiwin32"
+    ],
+        cwd=third_party_path,
+        merge_env=pip_env)
```

Comment:
Are we going to install yapf this way too?

Reply:
If you want to. It'd be trivial to do.  But Google storage seems to work so there's no rush.

Reply:
google storage?
I think we should do it since we have this infrastructure now.